hazelcast-sql/src/main/java/com/hazelcast/sql/impl/schema/view/View.java

```diff
@@ -73,6 +75,10 @@ public List<QueryDataType> viewColumnTypes() {
         return viewColumnTypes;
     }
 
+    public boolean isStreaming() {
```

Comment:
View computation process must not fully rely on data stored in `View`, especially on `isStreaming` method. Previously, we had a problem with streaming source detection, and when we solve that problem, we removed `isStream` property from `View` : https://github.com/hazelcast/hazelcast/pull/20996, https://github.com/hazelcast/hazelcast/pull/21028
 I'd suggest you write documentation for this method to forbid its usages as a source of truth for view computation.

Reply:
I've removed this field completely: bf9c2cc6ec4e4be6414e049dac0cc5ef419a8a5a